install.sh

```diff
@@ -53,6 +53,8 @@ CATKIN_DIR=~/mil_ws
 MIL_CONFIG_DIR=~/.mil
 BASHRC_FILE=~/.bashrc
 MILRC_FILE=$MIL_CONFIG_DIR/milrc
+BVTSDK_DIR=$MIL_CONFIG_DIR
+BVTSDK_DIR_COMPLETE=$BVTSDK_DIR/bvtsdk
```

Comment:
Why create a new variable just to mirror `MIL_CONFIG_DIR`? I think it world be better to just use `BVTSDK_DIR` for what `BVTSDK_DIR_COMPLETE` is being used for and then remove `BVTSDK_DIR_COMPLETE`.